### DISCUSSION TITLE
Regarding the problem of http 206 Content-Range error when the recorded file (fmp4) is played in the browser

### DISCUSSION THREAD

**Discussion by klk2** [bluenviron/mediamtx#2632] [Questions]
### Question

Using the record attribute to record fmp4 file, there seems to be a file size problem, which causes the wrong file size to be obtained when playing it in the browser. When playing it using the http Content-Range breakpoint, the entire file is obtained at once.

I use gin StaticFS and Static to play mp4 files

<img width="883" alt="image" src="https://github.com/bluenviron/mediamtx/assets/19720619/2d2f9805-1682-4366-b2e5-d6393982822f">
<img width="451" alt="image" src="https://github.com/bluenviron/mediamtx/assets/19720619/4e66aea8-e622-4670-ac23-7b6a1009d5c0">

This problem does not exist when using the browser to play the new MP4 file after conversion through ffmpeg.
It seems that the problem points to the mp4 itself and the recording

<img width="857" alt="image" src="https://github.com/bluenviron/mediamtx/assets/19720619/b8feda9d-c12b-4912-a1cd-05ad6230f554">

<img width="489" alt="image" src="https://github.com/bluenviron/mediamtx/assets/19720619/ad7ff72d-f8ee-45e0-ac2f-72c1808c25e9">

**Comment by 0ip** [comment]
Can you ffprobe your affected recordings and see if the `duration` is correctly set for them?

**Comment by aler9** [comment]
<s>Hello, as @0ip already suggested, this may have been caused by #2477, that was fixed by #2715. Please try again with the latest server version and let me know if recordings generated with it cause the same browser error.</s>

**Reply by fbotp** [reply]
Today, this question still exists.😟

**Reply by aler9** [reply]
@fbotp false. Use the embedded playback server to serve recording segments (do not serve them with a plain HTTP server) and append the `format=mp4` parameter as written in the README:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#playback-recorded-streams
